cypress/support/fragments/users/userEdit.js

```diff
@@ -258,4 +265,49 @@ export default {
   selectSingleSelectValue: ({ data }) => {
     cy.do(Select({ label: data.fieldLabel }).choose(data.firstLabel));
   },
+
+  verifyUserPermissionsAccordion() {
+    cy.expect(permissionsAccordion.exists());
+    cy.expect(permissionsAccordion.has({ open: false }));
```

Comment:
if we have more than one assertion in a row, better to pass them as an array

Reply:
Only one assertion here, the accordion is present and collapsed by default

Reply:
i meant like 
`    cy.expect([
      permissionsAccordion.exists(),
      permissionsAccordion.has({ open: false })
    ]);`